modules/openapi-generator/src/main/resources/python/model_templates/classvars.mustache

```diff
@@ -92,24 +92,24 @@
         """
 {{#imports}}
 {{#-first}}
-        lazy_import()
+        class_name_to_class = lazy_import()
```

Comment:
This line and its buddies dont get created if there are no imports obviously. And further down in 100, 103 etc, when I try to use the calling class' name, the dict is non-existent since 95 is not there.

Reply:
That is expected. Are there models that are missing imports?